src/eko/evolution_operator/__init__.py

```diff
@@ -11,15 +11,14 @@
 
 import numba as nb
 import numpy as np
-from scipy import integrate
-
 from .. import anomalous_dimensions as ad
 from .. import basis_rotation as br
 from .. import interpolation, mellin
 from .. import scale_variations as sv
 from ..kernels import non_singlet as ns
 from ..kernels import singlet as s
 from ..member import OpMember
+from scipy import integrate
```

Comment:
`pre-commit` (i.e. `isort`) should put it back in the former place, please run:
```sh
pre-commit install
```
in the local instance of your repository.